test/empty-keys-cases.js

```diff
@@ -2,36 +2,36 @@
 
 module.exports = {
     emptyTestCases: [
-        { input: '&', withEmptyKeys: {}, stringifyOutput: '', noEmptyKeys: {} },
-        { input: '&&', withEmptyKeys: {}, stringifyOutput: '', noEmptyKeys: {} },
-        { input: '&=', withEmptyKeys: { '': '' }, stringifyOutput: '=', noEmptyKeys: {} },
-        { input: '&=&', withEmptyKeys: { '': '' }, stringifyOutput: '=', noEmptyKeys: {} },
-        { input: '&=&=', withEmptyKeys: { '': ['', ''] }, stringifyOutput: '[0]=&[1]=', noEmptyKeys: {} },
-        { input: '&=&=&', withEmptyKeys: { '': ['', ''] }, stringifyOutput: '[0]=&[1]=', noEmptyKeys: {} },
+        { input: '&', withEmptyKeys: {}, stringifyOutput: '', noEmptyKeys: {}, stringifyOutputNoEmpty: '' },
+        { input: '&&', withEmptyKeys: {}, stringifyOutput: '', noEmptyKeys: {}, stringifyOutputNoEmpty: '' },
+        { input: '&=', withEmptyKeys: { '': '' }, stringifyOutput: '=', noEmptyKeys: {}, stringifyOutputNoEmpty: '' },
+        { input: '&=&', withEmptyKeys: { '': '' }, stringifyOutput: '=', noEmptyKeys: {}, stringifyOutputNoEmpty: '' },
+        { input: '&=&=', withEmptyKeys: { '': ['', ''] }, stringifyOutput: '[0]=&[1]=', noEmptyKeys: {}, stringifyOutputNoEmpty: '' },
+        { input: '&=&=&', withEmptyKeys: { '': ['', ''] }, stringifyOutput: '[0]=&[1]=', noEmptyKeys: {}, stringifyOutputNoEmpty: '' },
```

Comment:
it makes me *very* nervous to be modifying test cases for a bug fix. is there any chance for a default algorithm to infer `stringifyOutputNoEmpty` rather than needing to add it to each case?

Reply:
🤷 I think explicit cases are better here. I assume you thought that we should default to `''` if you want a smaller diff, then it also makes to do the same for `stringifyOutput`?